planner.rb

```diff
@@ -76,24 +79,38 @@ def business_days_left_in_sprint(date)
   days = business_days_between(date, sprint_end)
   case days
   when 0
-    "last work day of sprint"
+    #"last day of sprint"
+    I18n.t('last_sprint_day')
   when 1
-    "1 work day left in sprint"
+    #"1 day left in sprint"
+    I18n.t('one_day_left_in_sprint')
   else
-    "#{days} work days left in sprint"
+    #"#{days} days left in sprint"
+    I18n.t('days_left_in_sprint', count: days)
   end
 end
 
 def quarter(date)
   QUARTERS_BY_MONTH[date.month]
 end
 
+# pick summer or winter semester depending on the month
+def semester_year(date)
+  if date.month >= SUMMER_SEMESTER_START && date.month < WINTER_SEMESTER_START
+    res = I18n.l(date, format: :year)
+  else
+    res = I18n.l(date, format: :year) << " / " << I18n.l(date.next_year, format: :year)
+  end
+  res
```

Comment:
Since it sounded like you're new to ruby I hope you don't mind if I point out some simplifications. The language has an implicit return, meaning it will return the result of the last expression evaluated in the method. And likewise the if block has an implicit return. So this could become:
```
  if date.month >= SUMMER_SEMESTER_START && date.month < WINTER_SEMESTER_START
    I18n.l(date, format: :year)
  else
    I18n.l(date, format: :year) << " / " << I18n.l(date.next_year, format: :year)
  res
```
The other change I'd make is to use string interpolation rather:
```suggestion
  if date.month >= SUMMER_SEMESTER_START && date.month < WINTER_SEMESTER_START
    I18n.l(date, format: :year)
  else
    "#{I18n.l(date, format: :year)} / #{I18n.l(date.next_year, format: :year)}"
  res
```

I'm wondering if we could make some of these helper methods into procs so they could be defined in the config file and easier to customize.